pkg/cmd/roachtest/testdata/regression.diffs

```diff
@@ -78313,9 +78652,52 @@ diff -U3 --label=/mnt/data1/postgres/src/test/regress/expected/triggers.out --la
  AS 'begin RETURN NULL; end';
  CREATE TRIGGER no_op_trig INSTEAD OF INSERT OR UPDATE OR DELETE
  ON european_city_view FOR EACH ROW EXECUTE PROCEDURE no_op_trig_fn();
-+ERROR:  unimplemented: CREATE TRIGGER
-+HINT:  You have attempted to use a feature that is not yet implemented.
-+See: https://go.crdb.dev/issue-v/126359/_version_
++ERROR:  internal error: building declarative schema change targets for CREATE TRIGGER: runtime error: invalid memory address or nil pointer dereference
```

Comment:
I think this bug has already been addressed in #135667, but we can update the diff on the next failure.